### PR TITLE
Add Store/Remove transaction

### PR DESCRIPTION
**Pull request by sgkim126**

### CODE REVIEW

File: test/store-and-remove.spec.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+let signedStore: SignedTransaction;
+let signedRemove: SignedTransaction;
+
+beforeAll(async done => {
+    await Helper.runExample("store-and-remove");
+
+    const bestBlockNumber = await Helper.sdk.rpc.chain.getBestBlockNumber();
+    storeBlockNumber = bestBlockNumber - 1;
+    removeBlockNumber = bestBlockNumber;
+    const mintBlockResponse = await Helper.sdk.rpc.chain.getBlock(
```
Comment (xpdlf1004) [inline test/store-and-remove.spec.ts]: mintBlockResponse => storeBlockResponse

File: test/store-and-remove.spec.ts [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        storeBlockNumber
+    );
+    const transferBlockResponse = await Helper.sdk.rpc.chain.getBlock(
```
Comment (xpdlf1004) [inline test/store-and-remove.spec.ts]: transferBlockResponse -> removeBlockResponse

File: test/store-and-remove.spec.ts (hunk 1 above)
Comment (sgkim126) [reply test/store-and-remove.spec.ts]: Thank you.

**Comment by xpdlf1004** [comment]
LGTM except for some typo.

**Comment by sgkim126** [comment]
@xpdlf1004 
I checked tests passes on my machine several times, but Travis continuously failed.
Do you have any guess?

**Pull request merged**